CMakeLists.txt

```diff
@@ -364,6 +364,9 @@ cmake_dependent_option(IREE_TARGET_BACKEND_VULKAN_SPIRV "Enables the 'vulkan-spi
 # Default target backends that are not yet fully supported but are being brought up.
 cmake_dependent_option(IREE_TARGET_BACKEND_ROCM "Enables the 'rocm' compiler target backend" ${IREE_TARGET_BACKEND_DEFAULTS} ${IREE_BUILD_COMPILER} OFF)
 
+set(IREE_ROCM_PATH_DEFAULT "/opt/rocm")
+set(IREE_ROCM_PATH "${IREE_ROCM_PATH_DEFAULT}" CACHE STRING "Path to ROCm installation.")
```

Comment:
We should be removing this, not promoting it to the top level CMake file. See https://github.com/openxla/iree/pull/16803 for context.

(This might be fine on the `sdxl` branch, but we should watch for it landing on `main`)

Reply:
I think if just wanting to override the device bitcode with a custom version, there is already an option for that: https://github.com/openxla/iree/blob/2eae9b3e3d515ba92048319d2a3616f947fc095d/compiler/plugins/target/ROCM/CMakeLists.txt#L82

That just switches where it copies them from as it integrates them into the compiler vs changing the entire tooling flow.

Reply:
OK, but without it and if you have ROCm in a non-default path it does not compile.

Reply:
We just have had always ROCm at `/opt/rocm` and that is why people where not complaining.
Maybe the compiler can be made to not need it. But now it complains without it.

Reply:
Details?

There were many complaints with it as it was. It should work as it is now and I need error logs: outside of the ukernels, there should not be a dependency in the compiler in having rocm installed.

Reply:
Sorry for the late reply. It may be so that it is not need it anymore. I am closing this. Let's see if we experience problems after sdxl is merged into main.